spec/models/user_spec.rb

```diff
@@ -334,6 +334,24 @@
       end
     end
 
+    context "when name contains HTML tags" do
+      it "sanitizes HTML tags from name" do
+        user = create(:user, name: "<script>alert('xss')</script>Test name")
+        expect(user.display_name).to eq "Test name"
```

Comment:
We cannot create that user because validations will prevent it, will have to create it this way for test:

```suggestion
        user = build(:user, name: "<script>alert('xss')</script>Test name")
        user.save!(validate: false)
        expect(user.display_name).to eq "alert('xss')Test name"
```